src/main/webapp/WEB-INF/tags/instructor/course/courseEditAddInstructorPanel.tag

```diff
@@ -46,13 +46,12 @@
                 <div id="accessControlEditDivForInstr${addInstructorPanel.index}">
                     <div class="form-group">
                         <label class="col-sm-3 control-label">
-                            <input type="checkbox" name="<%=Const.ParamsNames.INSTRUCTOR_IS_DISPLAYED_TO_STUDENT%>" value="true" checked
-                               data-toggle="tooltip" data-placement="top" title="<%=Const.Tooltips.INSTRUCTOR_DISPLAYED_TO_STUDENT%>">
+                            <input id="displayToStudent" type="checkbox" name="<%=Const.ParamsNames.INSTRUCTOR_IS_DISPLAYED_TO_STUDENT%>" value="true" data-toggle="tooltip" data-placement="top" title="<%=Const.Tooltips.INSTRUCTOR_DISPLAYED_TO_STUDENT%>">
                             Display to students as:
                         </label>
                         <div class="col-sm-9">
-                            <input class="form-control" type="text" name="<%=Const.ParamsNames.INSTRUCTOR_DISPLAY_NAME%>"
-                                placeholder="E.g.Co-lecturer, Teaching Assistant"
+                            <input class="form-control" id="titleDisplayed" type="text" name="<%=Const.ParamsNames.INSTRUCTOR_DISPLAY_NAME%>"
+                                placeholder="Instructor" disabled
```

Comment:
The use of `placeholder` relies on the default behaviour that "Instructor" is automatically filled in if a blank name is submitted. It would be better to use `value` for this.

On the same line: use `disabled=""` to be consistent with jQuery behaviour.